core/src/test/scala/com/yahoo/maha/core/query/DefaultQueryPipelineFactoryTest.scala

```diff
@@ -708,16 +714,15 @@ class DefaultQueryPipelineFactoryTest extends FunSuite with Matchers with Before
         row.addValue("Ad Group ID", 10)
         row.addValue("Impressions", 100)
         row.addValue("Clicks", 1)
+        row.addValue("Pricing Type", "CPC")
         rl.addRow(row)
     }.withOracleCallback {
       rl =>
         val row = rl.newRow
         row.addValue("Advertiser ID", 1)
         row.addValue("Ad Group Status", "ON")
         row.addValue("Ad Group ID", 10)
-        row.addValue("Source", 2)
```

Comment:
Again, this test is for dim driven query.  You should not be modifying this test.  It works as expected.